client/lib/domains/google-apps-users/index.js

```diff
@@ -28,12 +28,25 @@ export function validate( { users, fields } ) {
 					error = i18n.translate( "This field can't be longer than 60 characters." );
 				}
 			} else if ( includes( [ 'email', 'username' ], key ) ) {
+				const newEmail = `${ field.value }@${ user.domain.value }`;
+
 				if ( ! /^[0-9a-z_'-](\.?[0-9a-z_'-])*$/i.test( field.value ) ) {
 					error = i18n.translate(
 						'Only number, letters, dashes, underscores, apostrophes and periods are allowed.'
 					);
-				} else if ( ! emailValidator.validate( `${ field.value }@${ user.domain.value }` ) ) {
+				} else if ( ! emailValidator.validate( newEmail ) ) {
 					error = i18n.translate( 'Please provide a valid email address.' );
+				} else if ( null !== existingUsers ) {
+					if (
+						includes(
+							mapValues( existingUsers, function( existingUser ) {
+								return existingUser.email;
+							} ),
+							newEmail
+						)
+					) {
+						error = i18n.translate( 'You already have this email address.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'You already have this email address!' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).